rules/imports.js

```diff
@@ -0,0 +1,283 @@
+
+    // Forbid import of modules using absolute paths
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-absolute-path.md
+    'import/no-absolute-path': ['error'],
+
+    // Forbid require() calls with expressions
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-dynamic-require.md
+    'import/no-dynamic-require': ['error'],
+
+    // prevent importing the submodules of other modules
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-internal-modules.md
+    'import/no-internal-modules': [
+      'off',
+      {
+        allow: [],
+      },
+    ],
+
+    // Warn if a module could be mistakenly parsed as a script by a consumer
+    // leveraging Unambiguous JavaScript Grammar
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/unambiguous.md
+    // this should not be enabled until this proposal has at least been *presented* to TC39.
+    // At the moment, it's not a thing.
+    'import/unambiguous': ['off'],
+
+    // Forbid Webpack loader syntax in imports
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-webpack-loader-syntax.md
+    'import/no-webpack-loader-syntax': ['error'],
+
+    // Prevent unassigned imports
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unassigned-import.md
+    // importing for side effects is perfectly acceptable, if you need side effects.
+    'import/no-unassigned-import': ['off'],
+
+    // Prevent importing the default as if it were named
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-named-default.md
+    'import/no-named-default': ['error'],
+
+    // Reports if a module's default export is unnamed
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-anonymous-default-export.md
+    'import/no-anonymous-default-export': [
+      'off',
+      {
+        allowArray: false,
+        allowArrowFunction: false,
+        allowAnonymousClass: false,
+        allowAnonymousFunction: false,
+        allowLiteral: false,
+        allowObject: false,
+      },
+    ],
+
+    // This rule enforces that all exports are declared at the bottom of the file.
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/exports-last.md
+    'import/exports-last': ['off'],
+
+    // Reports when named exports are not grouped together in a single export declaration
+    // or when multiple assignments to CommonJS module.exports or exports object are present
+    // in a single file.
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/group-exports.md
+    'import/group-exports': ['off'],
+
+    // forbid default exports. this is a terrible rule, do not use it.
+    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-default-export.md
+    'import/no-default-export': ['error'],
```

Comment:
[IMO] We should disable this rule for now. I’d not like to use this rule (as just the above comment says it's a terrible rule). [There are many situations using default exports](https://github.com/search?q=org%3Amoneyforward%20%22export%20default%22&type=code). It's too strict for our codebases. I'm afraid that enabling this rule will prevent people from using this package itself.

Reply:
@taigakiyokawa 
I'll enable this rule.

In application code, `default export` has many disadvantages such as compromising code consistency, so I belive that it should not be used in principle.

ESLint Config is a tool that users can tune and optimize individually. Our service platform is also developed using the same philosophy. Therefore, all users aren't forced to follow this rule.
Also, I don't intend to save all users with this ESLint Config. If we aim for that, we'll continue to create technical debt indefinitely.

However, it is only disabled in cases where libraries such as Storybook and Next.js require “default export”. They are implemented in #202, #205.

Reply:
OK, I got your point. Let's enable it.
However, you should remove the original comment above ("this is a terrible rule, do not use it."). It's inconsistent with your change.

```diff
-     // forbid default exports. this is a terrible rule, do not use it.
+    // forbid default exports.
```

Reply:
fix: [34dbc7a](https://github.com/moneyforward/eslint-config-moneyforward/pull/213/commits/34dbc7a86e6de89d7e1b38cc9fca88e044b0c12c)